roofit/hs3/src/JSONFactories_HistFactory.cxx

```diff
@@ -560,18 +560,13 @@ class FlexibleInterpVarStreamer : public RooFit::JSONIO::Exporter {
    }
    bool exportObject(RooJSONFactoryWSTool *, const RooAbsArg *func, JSONNode &elem) const override
    {
-      const RooStats::HistFactory::FlexibleInterpVar *fip =
-         static_cast<const RooStats::HistFactory::FlexibleInterpVar *>(func);
+      auto fip = static_cast<const RooStats::HistFactory::FlexibleInterpVar *>(func);
       elem["type"] << key();
-      auto &vars = elem["vars"];
-      elem["interpolationCodes"] << fip->interpolationCodes();
-      vars.set_seq();
-      for (const auto &v : fip->variables()) {
-         vars.append_child() << v->GetName();
-      }
+      elem["vars"].fill_seq(fip->variables(), [](auto const &item) { return item->GetName(); });
+      elem["interpolationCodes"].fill_seq(fip->interpolationCodes());
       elem["nom"] << fip->nominal();
-      elem["high"] << fip->high();
-      elem["low"] << fip->low();
+      elem["high"].fill_seq(fip->high());
```

Comment:
Why are you using `fill_seq`  instead of operator `<<` ?

Reply:
1. To make the code more readable (`fill_seq` is less implicit than operator <<, you know now that "high" is a list without even knowing the type of `fip->high()`.
2. The make the creation of lists more consistent: `fill_seq(coll, func)` is used to create a list in the JSON by transforming every element in `coll` with `func`, and you use `fill_sequ(coll)` when you want to do so with the default identify transformation